### ISSUE TITLE
no character limit on room names

### ISSUE THREAD

**Issue by warandpeace** [amirrajan/nodekick#37]

**Comment by jeremyjackson89** [comment]
So, room names should or shouldn't have a character limit?

**Comment by amirrajan** [comment]
I think he means that there _should_ be a character limit (currently there isn't one)

**Comment by jeremyjackson89** [comment]
Cool, I'll handle that later tonight unless someone else does first.

**Comment by warandpeace** [comment]
There probably should be, but it's @amirrajan's project. I logged it in the QA spirit of 'seems weird, possibly an issue?' but if you wanna support unlimited character room names, I ain't gonna complain 😛

**Comment by jeremyjackson89** [comment]
Yeah. I'll submit a pull request for it here in a bit and if @amirrajan wants to keep unlimited character room names, he can just not pull it in. :)
